### PR TITLE
Feat/4000 button colors

### PR DESCRIPTION
**Pull request by 328d95**

### CODE REVIEW

**Review by EduardoUmpierre — APPROVED** [review]
lgtm!

**Comment by gateixeira** [comment]
@328d95 can you please resolve the conflicts and update this PR so that we can close it? Been here for quite some time now.

File: src/components/Icons/FarmerBook2/index.tsx [hunk 1]
```diff
@@ -3,12 +3,10 @@ import * as React from 'react';
 const FarmerBook2: React.FC<React.SVGProps<SVGSVGElement>> = ({ ...props }) => (
```
Comment (EduardoUmpierre) [inline src/components/Icons/FarmerBook2/index.tsx:3]: Should we use the fill in the same way as the previous ones?

File: src/components/Icons/Report/index.tsx [hunk 2]
```diff
@@ -3,37 +3,30 @@ import * as React from 'react';
 const Report: React.FC<React.SVGProps<SVGSVGElement>> = ({ ...props }) => (
```
Comment (EduardoUmpierre) [inline src/components/Icons/Report/index.tsx:3]: I think the fill is missing here too

File: src/components/Icons/SMS/index.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+const SMS: React.FC<React.SVGProps<SVGSVGElement>> = ({ fill, ...props }) => (
```
Comment (EduardoUmpierre) [inline src/components/Icons/SMS/index.tsx:3]: Maybe we should provide a default value to `fill`

File: src/components/DatePicker/index.test.tsx [hunk 4]
```diff
@@ -7,34 +7,48 @@ describe('DatePicker component', () => {
...
+    });
+
+    fireEvent.click(container.getByLabelText('Choose Saturday, January 2nd, 2021'));
+    await waitFor(() => {
+      expect(onChange).toHaveBeenCalledWith(targetDate, targetDate);
+    });
   });
 
-  it('calls onChange with the correct arguments for date range selection', () => {
+  it('calls onChange with the correct arguments for date raange selection', async () => {
```
Comment (EduardoUmpierre) [inline src/components/DatePicker/index.test.tsx:34]: There's a typo on `range`

**Review by EduardoUmpierre — APPROVED** [review]
LGTM! Awesome work 👏

**Pull request merged**